models/workitem_repository_blackbox_test.go

```diff
@@ -0,0 +1,80 @@
+package models_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/almighty/almighty-core/app"
+	"github.com/almighty/almighty-core/criteria"
+	"github.com/almighty/almighty-core/models"
+	uuid "github.com/satori/go.uuid"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCreateLoadDeleteListWI(t *testing.T) {
+	doWithTransaction(t, func(ts *models.GormTransactionSupport) {
```

Comment:
- Hard  to reason what the test is about
- It tests too many things at once - as method already suggests.
- If something stops working you will go through snow-ball trying to fix every single failing assertion one after the other

Is it good enough or still not convinced?